src/libs/ReportUtils.js

```diff
@@ -1812,6 +1834,23 @@ function getWhisperDisplayNames(participants) {
     return _.map(participants, login => getDisplayNameForParticipant(login, !isWhisperOnlyVisibleToCurrentUSer)).join(', ');
 }
 
+/**
+ * Show subscript on IOU or expense report
+ * @param {Object} report
+ * @returns {Boolean}
+ */
+function shouldReportShowSubscript(report) {
+    if (isArchivedRoom(report)) {
+        return false;
+    }
```

Comment:
I believe this also applied to expense report, so I'm moving the check for both policy expense and expense report.